### PR TITLE
Feat/integração front back cadastro login

### PR DESCRIPTION
**Pull request by beviggiano**
This pull request integrates the frontend with the backend, incorporating authentication logic for the login and signup pages. The integration ensures seamless communication between the user interface and the server-side functionalities, enhancing the overall user experience and security of the application.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ export const App = () => {
         <>
             <BrowserRouter>
                 <Routes>
-                    <Route path="/" element={<Login/>} />
+                    <Route path="/login" element={<Login/>} />
                     <Route path ="/signup" element= {<SignUp/>} />
                     {/*<Route path="/posts" element={<Posts/>} />*/}
-                    <Route path="/posts" element={<AdvertisementScreen/>} />
```
Comment (edu-correia) [inline frontend/src/App.tsx:20]: No need to change this line

File: frontend/src/hooks/useLogin.ts
Comment (edu-correia) [inline frontend/src/hooks/useLogin.ts:1]: You didn't have to create a hook, but that will for sure help 👏

File: frontend/src/hooks/useSignUp.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import axios from "axios";
+
+const useSignUp = () => {
+    const [nome, setNome] = useState("");
+    const [email, setEmail] = useState("");
```
Comment (edu-correia) [inline frontend/src/hooks/useSignUp.ts:6]: let's use name and setName

File: frontend/src/public/logoPetAdopt.png:Zone.Identifier
Comment (edu-correia) [inline frontend/src/public/logoPetAdopt.png:Zone.Identifier:1]: Please remove this file

**Review by edu-correia — CHANGES_REQUESTED** [review]
Login flow looks good to me, let's just make the requested changes and we can proceed with the merge

File: frontend/src/hooks/useSignUp.ts (hunk 2 above)
Comment (beviggiano) [reply frontend/src/hooks/useSignUp.ts:6]: Just dit it

**Review by edu-correia — APPROVED** [review]
Approved 👍

**Pull request merged**